### PR TITLE
[SG-35282]: Wildcard V2: Apply <Input /> Codemod on `client/web/enterprise`

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Problem statement

Migration is required to update our code to use the `<Input />` Wildcard component.
See [parent issue](https://github.com/sourcegraph/sourcegraph/issues/27722) for more context. 

## Success criteria

All occurrences of the `<input />` element are replaced with the `<Input />` Wildcard component.

## Implementation details

- Rename the `Input` component `variant` prop to `size`.
- Use the `Input` codemod created [here](https://github.com/sourcegraph/codemod/pull/94) to speed up the migration.

## Refs
[Sourcegraph Issue](https://github.com/sourcegraph/sourcegraph/issues/35282)
[Gitstart Ticket](https://app.gitstart.com/clients/sourcegraph/tickets/SG-35282)

## Test Plan
- All `<input/>` elements are migrated to `<Input/>` wildcard component.
- No UI diff available

## App preview:

- [Web](https://sg-web-contractors-sg-35282.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-cmgcrwqaez.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

File: client/web/src/enterprise/search/stats/SearchStatsPage.tsx [hunk 1]
```diff
@@ -63,25 +63,26 @@ export const SearchStatsPage: React.FunctionComponent<React.PropsWithChildren<Pr
...
+                    inputClassName="flex-1"
+                    type="search"
+                    placeholder="Enter a Sourcegraph search query"
+                    value={uncommittedQuery}
+                    onChange={onUncommittedQueryChange}
+                    autoCapitalize="off"
+                    spellCheck={false}
+                    autoCorrect="off"
+                    autoComplete="off"
+                    inputSymbol={
```
Comment (valerybugakov) [inline client/web/src/enterprise/search/stats/SearchStatsPage.tsx:78]: Move button our of the `inputSymbol`

File: client/web/src/enterprise/insights/components/form/form-radio-input/FormRadioInput.tsx [hunk 2]
```diff
@@ -29,7 +29,7 @@ export const FormRadioInput: React.FunctionComponent<React.PropsWithChildren<Rad
                 'text-muted': otherProps.disabled,
             })}
         >
-            <input type="radio" {...otherProps} />
+            <Input formControl={false} type="radio" {...otherProps} />
```
Comment (valerybugakov) [inline client/web/src/enterprise/insights/components/form/form-radio-input/FormRadioInput.tsx:32]: Let's remove the `formControl` prop from the `Input` component and ignore the eslint rule here.

**Review by valerybugakov — CHANGES_REQUESTED** [review]

File: client/web/src/enterprise/codeintel/configuration/components/ReposMatchingPatternList.tsx [hunk 3]
```diff
@@ -42,28 +42,26 @@ export const ReposMatchingPatternList: FunctionComponent<React.PropsWithChildren
                 </div>
             ) : (
                 <>
-                    <div className={styles.grid}>
```
Comment (valerybugakov) [inline client/web/src/enterprise/codeintel/configuration/components/ReposMatchingPatternList.tsx:45]: Trying to understand what the reason for changing the layout here was?

**Review by valerybugakov — CHANGES_REQUESTED** [review]

File: client/web/src/enterprise/codeintel/configuration/components/ReposMatchingPatternList.tsx (hunk 3 above)
Comment (gitstart-sourcegraph) [reply client/web/src/enterprise/codeintel/configuration/components/ReposMatchingPatternList.tsx:45]: Was changed initially to support usage of `inputSymbol` here, safe to revert now...

**Review by valerybugakov — APPROVED** [review]
Thanks!

**Pull request merged**